AutomatedTesting/src/test/resources/features/AddPlanetUniquename.feature

```diff
@@ -0,0 +1,10 @@
+Scenario Outline: Add Planet to Planetarium  Unique name Test Case
```

Comment:
Feature Files start with "Feature: ..." on the first line and "Scenario Outline: ..." on the following line indented.